From a0dbb7774735928b80f6c24e4edca4eb3f336c73 Mon Sep 17 00:00:00 2001
From: Mingcong Bai <baimingcong@uniontech.com>
Date: Mon, 27 May 2024 16:27:56 +0800
Subject: [PATCH 119/360] arch: loongarch: update defconfig

- Disable strict /dev/mem access as required by some vendor applications.
- Refresh arch/loongarch/deepin_loongarch_desktop_defconfig.

Signed-off-by: Mingcong Bai <baimingcong@uniontech.com>
---
 .../configs/deepin_loongarch_desktop_defconfig   | 16 +++++++---------
 1 file changed, 7 insertions(+), 9 deletions(-)

diff --git a/arch/loongarch/configs/deepin_loongarch_desktop_defconfig b/arch/loongarch/configs/deepin_loongarch_desktop_defconfig
index 1840a22e224b..4883b7ea1b62 100644
--- a/arch/loongarch/configs/deepin_loongarch_desktop_defconfig
+++ b/arch/loongarch/configs/deepin_loongarch_desktop_defconfig
@@ -52,7 +52,7 @@ CONFIG_PROFILING=y
 CONFIG_KEXEC=y
 CONFIG_CRASH_DUMP=y
 CONFIG_HZ_1000=y
-CONFIG_4KB_4LEVEL=y
+CONFIG_NR_CPUS=256
 CONFIG_NUMA=y
 CONFIG_ARCH_FORCE_MAX_ORDER=12
 CONFIG_ARCH_IOREMAP=y
@@ -60,6 +60,10 @@ CONFIG_CPU_HAS_LSX=y
 CONFIG_CPU_HAS_LASX=y
 CONFIG_CPU_HAS_LBT=y
 CONFIG_RANDOMIZE_BASE=y
+CONFIG_CPU_FREQ=y
+CONFIG_CPU_FREQ_STAT=y
+CONFIG_CPU_FREQ_GOV_POWERSAVE=y
+CONFIG_LOONGSON3_ACPI_CPUFREQ=y
 CONFIG_HIBERNATION=y
 CONFIG_PM_AUTOSLEEP=y
 CONFIG_PM_WAKELOCKS=y
@@ -70,10 +74,6 @@ CONFIG_ACPI_TAD=m
 CONFIG_ACPI_DOCK=y
 CONFIG_ACPI_IPMI=m
 CONFIG_ACPI_PCI_SLOT=y
-CONFIG_CPU_FREQ=y
-CONFIG_CPU_FREQ_STAT=y
-CONFIG_CPU_FREQ_GOV_POWERSAVE=y
-CONFIG_LOONGSON3_ACPI_CPUFREQ=y
 CONFIG_ACPI_HOTPLUG_MEMORY=y
 CONFIG_ACPI_HED=y
 CONFIG_ACPI_CUSTOM_METHOD=y
@@ -3626,7 +3626,6 @@ CONFIG_FB_ASILIANT=y
 CONFIG_FB_IMSTT=y
 CONFIG_FB_UVESA=m
 CONFIG_FB_EFI=y
-CONFIG_FB_LS2K500=m
 CONFIG_FB_OPENCORES=m
 CONFIG_FB_S1D13XXX=m
 CONFIG_FB_RIVA=m
@@ -3670,6 +3669,7 @@ CONFIG_FB_METRONOME=m
 CONFIG_FB_MB862XX=m
 CONFIG_FB_SSD1307=m
 CONFIG_FB_SM712=m
+CONFIG_FB_LS2K500=m
 CONFIG_FIRMWARE_EDID=y
 CONFIG_LCD_CLASS_DEVICE=m
 CONFIG_LCD_L4F00242T03=m
@@ -5622,7 +5622,6 @@ CONFIG_EXT2_FS_SECURITY=y
 CONFIG_EXT3_FS=m
 CONFIG_EXT3_FS_POSIX_ACL=y
 CONFIG_EXT3_FS_SECURITY=y
-CONFIG_EXT4_FS=y
 CONFIG_REISERFS_FS=m
 CONFIG_REISERFS_PROC_INFO=y
 CONFIG_REISERFS_FS_XATTR=y
@@ -5975,7 +5974,6 @@ CONFIG_FONT_6x8=y
 CONFIG_PRINTK_TIME=y
 CONFIG_DYNAMIC_DEBUG=y
 CONFIG_DEBUG_INFO_DWARF5=y
-CONFIG_DEBUG_INFO_BTF=y
 CONFIG_GDB_SCRIPTS=y
 CONFIG_STRIP_ASM_SYMS=y
 CONFIG_READABLE_ASM=y
@@ -6015,6 +6013,6 @@ CONFIG_TRACE_EVENT_INJECT=y
 CONFIG_RV=y
 CONFIG_RV_MON_WIP=y
 CONFIG_RV_MON_WWNR=y
-CONFIG_IO_STRICT_DEVMEM=y
+# CONFIG_STRICT_DEVMEM is not set
 CONFIG_FUNCTION_ERROR_INJECTION=y
 # CONFIG_RUNTIME_TESTING_MENU is not set
-- 
2.20.1

